omod/src/main/java/org/openmrs/module/coreapps/page/controller/findpatient/FindPatientPageController.java

```diff
@@ -28,6 +31,7 @@ public void get(PageModel model, @RequestParam("app") AppDescriptor app, UiSessi
         model.addAttribute("heading", app.getConfig().get("heading").getTextValue());
         model.addAttribute("label", app.getConfig().get("label").getTextValue());
         model.addAttribute("showLastViewedPatients", app.getConfig().get("showLastViewedPatients").getBooleanValue());
+		model.addAttribute("showRegistrationLink", app.getConfig().get("showRegistrationLink").getTextValue());
```

Comment:
Well, _showRegistrationLink_ gives the impression that it takes a boolean value, it would be nice to rename to some thing more intuitive that implies its an appId